dashboard/src/components/shared/format/Money.vue

```diff
@@ -25,6 +35,33 @@ export default class Money extends Vue {
     return this.chainProperties.tokenSymbol
   }
 
-}
+  public created() {
+    if (this.showFiatValue) {
+      this.getFiatValue()
+    }
+  }
+
+  private async getFiatValue() {
+    try {
+      const { data } = await axios.get(`${this.apiUrl}/simple/price`, {
```

Comment:
I would move this to separate file `src/utils/coingecko.ts` or `src/coingecko.ts` or under `src/fetch.ts`

Reply:
moved to `src/coingecko.ts` please check if it is that you want, I've added just new axios instance with default url to coingecko

Reply:
If I misunderstood it and you wanted to move whole request to separate file just let me know

Reply:
Yeah I just wanted to move it so separate file